verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -298,13 +360,13 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                                     isChecked:
                                         values.sex !== undefined ||
                                         (values.age !== undefined &&
-                                            values.age !== '') ||
+                                            values.age.toString() !== '') ||
```

Comment:
When is it not a string? i.e. why is `toString()` necessary?

Reply:
values.age is a number, so this check was always false. An eslint warning pointed it out.